cms/djangoapps/contentstore/views/videos.py

```diff
@@ -328,6 +331,13 @@ def videos_post(course, request):
     for req_file in req_files:
         file_name = req_file["file_name"]
 
+        # Check if file_name contains non-ascii characters.
+        try:
+            file_name.encode('ascii')
+        except UnicodeEncodeError as exception:  # pylint: disable=unused-argument
+            error_msg = "%s contains non ascii characters in file name." % file_name
```

Comment:
@catong SO to be clear, final message would be like :

> Q19_80’_Mantra.mp4 must contain only ASCII characters in file name.

I think, it is better to show user which file failed (in case multiple files were uploaded).